satpy/tests/reader_tests/gms/test_gms5_vissr_l1b.py

```diff
@@ -9,14 +9,19 @@
 import xarray as xr
 from pyresample.geometry import AreaDefinition
 
-import satpy.readers.gms.gms5_vissr_format as fmt
-import satpy.readers.gms.gms5_vissr_l1b as vissr
-import satpy.readers.gms.gms5_vissr_navigation as nav
 import satpy.tests.reader_tests.gms.test_gms5_vissr_data as real_world
 from satpy.readers import FSFile
-from satpy.tests.reader_tests.utils import get_jit_methods
+from satpy.tests.reader_tests.utils import get_jit_methods, skip_numba_unstable_if_missing
 from satpy.tests.utils import make_dataid
 
+try:
+    import satpy.readers.gms.gms5_vissr_format as fmt
+    import satpy.readers.gms.gms5_vissr_l1b as vissr
+    import satpy.readers.gms.gms5_vissr_navigation as nav
+except ImportError:
+    if skip_numba_unstable_if_missing():
+        pytest.skip("Numba is not compatible with unstable NumPy", allow_module_level=True)
+
```

Comment:
Two things:

1. What happens if `skip_numba_unstable_if_missing()` returns false? I suspect the tests will fail with nameerrors because the gms5 modules have not been imported. Could we instead fail the full module directly somehow? or just raise the error again?
2. To be sure about why the tests are skipped, could we include the ImportError message in the skip string?

Reply:
Good points. What if I added a `raise` if it returns ``False``? That should be equivalent to the old behavior. I can try adding the error message somewhere. What is the current best way to get the message from an exception? `str(err)`?

Reply:
I'm good with `raise`.
Afaik `str(err)` is the way to go yes.

Reply:
Done.